### ISSUE TITLE
Keras conversion for layers with padding

### ISSUE THREAD

**Issue by sdemarch** [NeVerTools/pyNeVer#14]
In Keras padding is represented as a separate layer, Convolutional and Pooling layers must implement this distinction

**Comment by sdemarch** [comment]
Convolutional layers are supported only with padding = 'valid' for the moment

**Comment by sdemarch** [comment]
Efforts for tensorflow support have been discarded
